Gemfile.lock

```diff
@@ -152,12 +152,13 @@ GEM
     lumberjack (1.0.9)
     mail (2.6.3)
       mime-types (>= 1.16, < 3)
-    mas-rad_core (0.0.17)
+    mas-rad_core (0.0.19)
       active_model_serializers
       geocoder
       http
       pg
       rails (~> 4.2.0)
+      redis
```

Comment:
redis is locked at 3.2.0, while on consumer it's 3.2.1, is it worth bumping as well, just for the sake of consistency?